pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -52,6 +55,8 @@ def __init__(
         validate_data: bool = True,
         control_columns: Optional[List[str]] = None,
         yearly_seasonality: Optional[int] = None,
+        adstock: Union[str, Any] = "geometric",
+        saturation: Union[str, Any] = "logistic",
```

Comment:
If we pass the class, let's pass an instance of the class. This would change the way the way the class is initialized though and it wouldn't take the model itself. 

With that, we can be more precise than `Any`

Reply:
So, we can left the class as it is but you are suggesting organise the code to use `CustomClass.apply` in the model instead of the class only? Okay!